.github/workflows/TEST.yml

```diff
@@ -0,0 +1,90 @@
+name: TEST
+
+on:
+  push:
+
+  schedule:
```

Comment:
why a wildcard schedule. This should be run every minute all day?

Reply:
Yes, just for testing purpose. post testing will remove this to 12:05 AM

Reply:
Removed every minute run